<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog2</class>
 <widget class="QWidget" name="dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1893</width>
    <height>1206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="pushButton_ajouter_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>521</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>                             Afficher les clients</string>
   </property>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>471</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>20</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/Artboard-35-ConvertImage.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>-50</y>
     <width>1981</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/15190.jpg</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>440</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Calculer les points de fidélité</string>
   </property>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Calculer les points de fidélité</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>540</y>
     <width>581</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Envoyer un mail de confirmation</string>
   </property>
   <widget class="QPushButton" name="pushButton_Mail">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="msg1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Objet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;À&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rcpt1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>471</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Ajouter un client</string>
   </property>
   <widget class="QPushButton" name="pushButton_Ajouter">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Quitter">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Id">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Prénom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Adresse e-mail&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;N° de téléphone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Adresse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Nom">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Prenom">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Email">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Numt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Adresse">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Point de fidélité&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Ptf">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Identifiant du client&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Trier les clients</string>
   </property>
   <widget class="QPushButton" name="Button_TriId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par identifiant</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_TriN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par nom</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_TriPt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par point de fidélité</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Rechercher les clients</string>
   </property>
   <widget class="QPushButton" name="Button_RechP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par prénom</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_RechN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par nom</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_RechNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par N° de téléphone</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Rnom1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Rprenom1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Rnum1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>60</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Modifier ou Supprimer un client</string>
   </property>
   <widget class="QPushButton" name="pushButton_Supprimer">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Modifier">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Ids">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Id&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>330</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Extaire PDF</string>
   </property>
   <widget class="QPushButton" name="pushButton_Extr">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Extraire</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView0">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>60</y>
     <width>251</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>ARDUINO</string>
   </property>
   <widget class="QPushButton" name="pushButton_refuser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>REFUSER ACCES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
gridline-color: rgb(0, 0, 0);
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acces">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>DONNER ACCES</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recource.qrc"/>
 </resources>
 <connections/>
</ui>
